lforms-converter.js

```diff
@@ -398,6 +398,9 @@ function doSkipLogic(root) {
 
   traverseItems(root, function(item, ancestors) {
     if(item.skipLogic) {
+      if (item.skipLogic.condition != undefined && item.skipLogic.condition.trim() === "") {
```

Comment:
Previously it is assumed that condition is always there in the skipLogic. But if you think it may not be there, 
don't you need to remove skipLogic, even when skipLogic.condition does not exist? I was expecting to see this...

if (item.skipLogic.condition === undefined || item.skipLogic.condition.trim() === "")